source/_integrations/androidtv_remote.markdown

```diff
@@ -75,99 +66,34 @@ Example:
 # Change channel to number 15:
 service: media_player.play_media
 data:
-  media_content_type: channel
+  media_content_type: "channel"
   media_content_id: 15
 target:
   entity_id: media_player.living_room_tv
 ```
 
-## Remote
+## Entity
```

Comment:
I think you want to rename this section to Remote now

Reply:
I just reverted the changes before #27059, since it was marked as unrelated.

Reply:
I wouldn't call this unrelated. The old documentation was structured for implementing a single platform. Since now it's 2 platforms you need to structure it accordingly and have 2 sections with subsections.

Reply:
Done